core/src/main/java/io/micronaut/core/beans/AbstractBeanIntrospection.java

```diff
@@ -1,356 +0,0 @@
-/*
- * Copyright 2017-2020 original authors
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * https://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package io.micronaut.core.beans;
-
-import io.micronaut.core.annotation.AnnotationMetadata;
-import io.micronaut.core.annotation.Internal;
-import io.micronaut.core.annotation.UsedByGeneratedCode;
-import io.micronaut.core.reflect.ReflectionUtils;
-import io.micronaut.core.reflect.exception.InstantiationException;
-import io.micronaut.core.type.Argument;
-import io.micronaut.core.util.ArgumentUtils;
-import io.micronaut.core.util.ObjectUtils;
-import io.micronaut.core.util.StringUtils;
-
-import io.micronaut.core.annotation.NonNull;
-import io.micronaut.core.annotation.Nullable;
-
-import java.lang.annotation.Annotation;
-import java.util.*;
-
-/**
- * Abstract implementation of the {@link BeanIntrospection} interface. This class is subclasses at compilation time by generated byte code and should not be used directly.
- *
- * @param <T> The generic type
- * @author graemerocher
- * @since 1.1
- */
-@UsedByGeneratedCode
-@Internal
-public abstract class AbstractBeanIntrospection<T> implements BeanIntrospection<T> {
```

Comment:
Is this file not needed any more?

Reply:
no the introspections have used `AbstractInitializableBeanIntrospection` for a while now